src/main/java/frc/robot/KrakenSwerveConfig.java

```diff
@@ -0,0 +1,102 @@
+        private static final int FRONT_RIGHT_ROT_ID = 3;
+        private static final int FRONT_RIGHT_ROT_ENC_ID = 23;
+
+        private static final int BACK_RIGHT_TRANS_ID = 9;
+        private static final int BACK_RIGHT_ROT_ID = 6;
+        private static final int BACK_RIGHT_ROT_ENC_ID = 20;
+
+        private static final int BACK_LEFT_TRANS_ID = 5;
+        private static final int BACK_LEFT_ROT_ID = 4;
+        private static final int BACK_LEFT_ROT_ENC_ID = 21;
+
+
+        public static final int PIGEON_ID = 25;
+
+
+        // Inverse Booleans
+        private static final boolean FRONT_LEFT_ROT_INVERSE = false;
+        private static final boolean FRONT_LEFT_TRANS_INVERSE = false;
+
+        private static final boolean FRONT_RIGHT_ROT_INVERSE = false;
+        private static final boolean FRONT_RIGHT_TRANS_INVERSE = true;
+
+        private static final boolean BACK_LEFT_ROT_INVERSE = false;
+        private static final boolean BACK_LEFT_TRANS_INVERSE = false;
+
+        private static final boolean BACK_RIGHT_ROT_INVERSE = false;
+        private static final boolean BACK_RIGHT_TRANS_INVERSE = true;
+
+        // Swerve Module Locations
+        public static final Translation2d FRONT_LEFT_COORDS_METERS =
+                        new Translation2d(SwerveConstants.TRACK_LENGTH_METERS / 2,
+                                        SwerveConstants.TRACK_WIDTH_METERS / 2);
+        public static final Translation2d FRONT_RIGHT_COORDS_METERS =
+                        new Translation2d(SwerveConstants.TRACK_LENGTH_METERS / 2,
+                                        -SwerveConstants.TRACK_WIDTH_METERS / 2);
+        public static final Translation2d BACK_LEFT_COORDS_METERS =
+                        new Translation2d(-SwerveConstants.TRACK_LENGTH_METERS / 2,
+                                        SwerveConstants.TRACK_WIDTH_METERS / 2);
+        public static final Translation2d BACK_RIGHT_COORDS_METERS =
+                        new Translation2d(-SwerveConstants.TRACK_LENGTH_METERS / 2,
+                                        -SwerveConstants.TRACK_WIDTH_METERS / 2);
+
+        // PID Controllers
+        public static final PIDController FRONT_LEFT_ROT_PID = new PIDController(0.35, 0., 0.);
+        public static final PIDController FRONT_RIGHT_ROT_PID = new PIDController(0.35, 0.0, 0.);
+        public static final PIDController BACK_RIGHT_ROT_PID = new PIDController(0.35, 0, 0.);
+        public static final PIDController BACK_LEFT_ROT_PID = new PIDController(0.35, 0, 0);
+
+
+        public static final SwerveDriveKinematics SWERVE_KINEMATICS = new SwerveDriveKinematics(
+                        FRONT_LEFT_COORDS_METERS, FRONT_RIGHT_COORDS_METERS,
+                        BACK_LEFT_COORDS_METERS, BACK_RIGHT_COORDS_METERS);
+
+
+        // Swerve Modules and Other Hardware
+        public static final SwerveModule FRONT_LEFT_MODULE = new SwerveModule(FRONT_LEFT_ROT_ID,
```

Comment:
do these need to be public? arent you going to access them via the SWERVE_MODULES array?